### DISCUSSION TITLE
Group Messaging

### DISCUSSION THREAD

**Discussion by kidder014** [markqvist/Reticulum#268] [Nomad Network]
Just started playing around with Reticulum and Nomad - have two Rnodes working between one another just fine, and Sideband as well.

It would be nice (although maybe technically challenging) to be able to send text messages via Nomad to a group simultaneously - essentially a "chat room" limited to a specific group of addresses. I have a group of 10 - 15 people that would like this functionality. Has there been any discussion along this route, or an option I'm not yet aware of?

**Comment by markqvist** [comment]
It's coming, but needs to be added to the underlying LXMF protocol. It is something that a lot of people want, but it will take a little time and effort to do it right, since it is non-trivial to create this sort of functionality, in a system that also needs to work efficiently over all kinds of low-bandwidth mediums.
